networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/P2PConfig.java

```diff
@@ -79,6 +97,9 @@ public int getPeerRequestLimit() {
     private final NetworkConfig.Builder networkConfig = NetworkConfig.builder();
     private final DiscoveryConfig.Builder discoveryConfig = DiscoveryConfig.builder();
 
+    private SpecConstants specConstants;
+    private Boolean isGossipScoringEnabled = false;
+    private GossipEncoding gossipEncoding = GossipEncoding.SSZ_SNAPPY;
```

Comment:
You mean we should be using a setter for this like: `config.gossipEncoding(GossipEncoding.SSZ_SNAPPY)`?  I had it set up like that at one point, but we only have the 1 option right now, so I thought it would be better to just leave the default value here.